src/main/scala-2.12/org/apache/james/gatling/jmap/JmapMessages.scala

```diff
@@ -111,6 +111,14 @@ object JmapMessages {
           "#0"
           ]]"""))
 
+  def listFirstNthMessages(numberOfMessages : Int) =
```

Comment:
N being 30 here (:

Reply:
Actually this method is not even used, i deleted it.